.env.test

```diff
@@ -0,0 +1,19 @@
+# Login details and Dev config for TFS API
+
+# App
+APP_ENV=test
+
+# Swagger credentials
+SWAGGER_USER=test-swagger-user
```

Comment:
we can add environment variables in Github and they will be loaded in the repository. We shouldn't need the test file

Reply:
If we do this then the developer would need to keep changing their `.env` file as they swap between running the api tests and running `start:dev`

Reply:
We're going to solve this with https://github.com/UK-Export-Finance/tfs-api/pull/11#discussion_r1126427712